modules/introduction/partials/new-features-76.adoc

```diff
@@ -158,6 +158,11 @@ For more information on the vector search, see xref:vector-search:vector-search.
 
 === Data Service
 
+* KV Range Scan is a method for accessing documents without necessarily knowing the IDs (keys).
```

Comment:
@RichardSmedley I chatted with Vishal about this--we iterated on the wording a bit and came up with: 

Introduces KV Range Scan, used  to retrieve all documents in a specified range directly from the Data service. Note that in this initial version, you will achieve better performance using a direct fetch or retrieval from a Query with an Index.

Reply:
Updated